examples/llama2/llama2_genai.json

```diff
@@ -0,0 +1,44 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",

```

Comment:
I think `PytorchModel` with just the `hf_config` would be okay here? We have multiple examples where we have a huggingface config only for input models like https://github.com/microsoft/Olive/blob/dd765a50ecb9e8f37745705d29ee60f50bf0c7ee/examples/bert/bert_cuda_gpu.json#L5. 
Since this is a model handler, the underlying pytorch nn.module is not loaded until `prepare_session()` is called. 

This is also one of the reasons we removed the `OptimumModelHandler` since it was just the role of a `PytorchModelHandler` with `hf_config`.

Reply:
ok

Reply:
Trimmed down the config to bear minimum required.